API.md

```diff
@@ -115,6 +115,32 @@ RNPdftron.encryptDocument("/sdcard/Download/new.pdf", "1111", "").then(() => {
 });
 ```
 
+### pdfFromOfficeTemplate
+Generates a PDF using a template in the form of an Office document and replacement data in the form of a JSON object.
+
+Parameters:
+
+Name | Type | Description
+--- | --- | ---
+docxPath | string | the local file path to the template file
+json | object | the replacement data in the form of a JSON object
+
+Returns a Promise.
+
+Promise Parameters:
+
+Name | Type | Description
+--- | --- | ---
+resultPdfPath | string | the local file path to the generated PDF (This file is temporary and must be handled by the user)
```

Comment:
- [ ] make it explicit that "user is responsible for cleaning up the temporary file"